include/ListView/ListViewSubPanel.php

```diff
@@ -417,9 +417,16 @@ public function process_dynamic_listview_rows($data, $parent_data, $smartyTempla
                                 $alias_field_def['name'] = $list_field['alias'];
                                 // Add alias field def into bean to can render field in subpanel
                                 $aItem->field_defs[$list_field['alias']] = $alias_field_def;
-                                if (!isset($fields[strtoupper($list_field['alias'])]) || empty($fields[strtoupper($list_field['alias'])])) {
+
+                                $fields[strtoupper($list_field['alias'])] = !empty($aItem->{$list_field['alias']})
+                                    ? $aItem->{$list_field['alias']}
+                                    : $aItem->$field_name;
+
+                                if (($alias_field_def['dbType'] ?? '') === 'datetime') {
```

Comment:
No Null coalescing operators until we're on PHP 7 :(

Reply:
True dat, fixed.